### PR TITLE
113-japanese-localization-old

### PR DESCRIPTION
**Pull request by shokokb**
Pull Request: 113-japanese-localization
File: ja.json

**Main working changes**
I don't rewrite sentences and just fixed some.
- スピーカー（speaker）->講演者（"Kouensha" - Speaker, Presenter, Teacher(Professors, etc) & Social influencers）
- スピーチ（speech）->講演（"Koen" - speak, make presentations & demonstrations）
- プレゼン（Presentation）-> 講演（"Koen" - speak, make presentations & demonstrations）
- 職名(Job Title) ->  役職 ex:President（社長）,Representative Director（代表取締役）,Chief Executive Officer / CEO（最高経営責任者）
- パネルディスカッション -> パネル・ディスカッション (add a dot to make it more readable)
- こちらのwebサイト->こちらのサイト (remove "web")
- **make long sentences shorter because they are easier to read**
- replace "よ**って**, し**て**" with "より,し" (sound officially on business)

```
-        "freeA": "このサイトを介して講演者に連絡することは無料です。しかし、スピーカによって有償の講演者もいれば、無償の講演者もいます。私たちは、講演者の考えを尊重し、有料であれば時間と労力に合う価格設定をサポートすることを検討しています。",
+        "freeA": "このサイトを介して講演者に連絡することは無料です。しかし、講演者により有償の場合、無償の場合があります。私たちは、講演者の考えを尊重し、有料であれば時間と労力に合う価格設定をサポートすることを検討しています。",
```

```
-        "freeQ": "スピーチは無料で行われるのでしょうか？",
+        "freeQ": "講演は無料で行われるのでしょうか？",
```

```
-            "hint": "何かを入力してEnterキーを押し、新しいトピックを追加します"
+            "hint": "分野を入力後、Enterキーを押し、新しい分野を追加します"
```

required = 必須（"Hissu")
```
-        "messageRequired": "スピーカーへの簡単なメッセージが必要です", (A brief message is required)
+        "messageRequired": "講演者への簡単なメッセージが必須です,
```

提供（"Teikyo" - provide） = 入力（"Nyuryoku" - input)
```
-        "valid": "有効なメールアドレスを提供してください"
+        "valid": "有効なメールアドレスを入力してください"
```

**Pronouns（敬称）**
We usually use only 様(sama) to a person
> Please type in pronouns if yours are not listed
> 敬称がない場合は、入力してください

**QA/Screenshots**
I yarn serve and check my Japanese ver.
screen-2020-09-06 20 29 28
screen-2020-09-06 20 29 48
screen-2020-09-06 20 32 12

### CODE REVIEW

**Comment by ann-kilzer** [comment]
Thanks, I have assigned @Ania-chan to review, as she is fluent.

Can you clarify your comment on pronouns?

> Pronouns（敬称）
> We usually use only 様(sama) to a person
> 
> Please type in pronouns if yours are not listed
> 敬称がない場合は、入力してください

We wish to know if a person uses "she" or "they" or [something else](https://pronouns.is). I suppose that would be 彼女、彼ら。。。 but I am not very good at Japanese, and I am unfamiliar with the LGBTQIA+ vocabulary here.

**Comment by shokokb** [comment]
> Thanks, I have assigned @Ania-chan to review, as she is fluent.
> 
> Can you clarify your comment on pronouns?
> 
> > Pronouns（敬称）
> > We usually use only 様(sama) to a person
> > Please type in pronouns if yours are not listed
> > 敬称がない場合は、入力してください
> 
> We wish to know if a person uses "she" or "they" or [something else](https://pronouns.is). I suppose that would be 彼女、彼ら。。。 but I am not very good at Japanese, and I am unfamiliar with the LGBTQIA+ vocabulary here.

We don't much care about he/she/they. Commonly, we only say "あなた(Anata, You)".
For example, "If you don't mind, let's keep in touch" 差し支えなければ、"あなた"と連絡を取りあいたいです。
I'm not sure that I understand what you want to do on the site with the pronouns.

**Comment by ann-kilzer** [comment]
I understand the Japanese language often omits the subject and implies things. However, especially in the English language, to be respectful of people with [non-binary genders](https://en.wikipedia.org/wiki/Non-binary_gender), we ask their preferred pronouns. For instance, mine are "she/her," but my non-binary friend takes "they/them," which is getting redefined in everyday English to be a 3rd person singular pronoun. This is a fairly new concept in Japan so we may need to ask some LGBTIA+ activists for the best translation.  We can skip translating it and just use the English fallback for now.

**Comment by ann-kilzer** [comment]
Actually this might be a useful resource: https://queerascat.tumblr.com/post/149940951871/how-do-i-tell-someone-im-non-binary-in-japanese

File: web/src/i18n/locales/ja.json [hunk 1]
```diff
@@ -5,11 +5,11 @@
         "title": "404：ページが見つかりません"
     },
     "about": {
-        "purpose": "登録者の情報は、イベントの企画者が女性スピーカーを簡単に見つけられるようにするため、また、ご登録いただいた方々が繋がって新しいコミュニティーを作るために使われます。ご注意 : Eメールアドレスを除き登録された情報はWebサイト上に公開されます。公開を望まない情報は共有しないでください。",
+        "purpose": "登録者の情報は、イベントの企画者が女性の講演者を簡単に見つけられるようにする目的に使用されます。ご登録いただいた方々がつながって新しいコミュニティーを作るために活用されます。ご注意 : Eメールアドレスを除き登録された情報はWebサイト上に公開されます。公開を望まない情報は共有しないでください。",
```
[the remark this answers is not in the document]
Comment (Ania-chan) [reply web/src/i18n/locales/ja.json:8]: Also if you think it's better we can remove `Web` from `Webサイト` in this sentence too

File: web/src/i18n/locales/ja.json [hunk 2]
```diff
@@ -20,7 +20,7 @@
     },
     "codeOfConduct": {
         "title": "行動規範",
-        "body": "本ウェブサイトは、女性スピーカーのアピールや募集以外の目的でを利用しないでください。私たちはコミュニティーの安全性を非常に重要視しています。こちらの目的以外の勧誘や嫌がらせはご遠慮ください。"
+        "body": "本ウェブサイトは、女性の講演者のアピールや募集以外の目的でを利用しないでください。私たちはコミュニティーの安全性を非常に重要視しています。こちらの目的以外の勧誘や嫌がらせはご遠慮ください。"
```
Comment (Ania-chan) [inline web/src/i18n/locales/ja.json:23]: I think it's a grammar mistake that I missed last time `でを`

File: web/src/i18n/locales/ja.json [hunk 3]
```diff
@@ -69,11 +69,11 @@
         },
         "city": "都市",
         "company": "会社",
-        "consent": "スピーカーの情報をSpeakHerデータベースに登録する許可をスピーカーから得ています。",
+        "consent": "講演者の情報をSpeakHerデータベースに登録する許可を講演者から得ています。",
         "email": "Eメール",
         "error": "エラー： {0}",
         "facebook": "FacebookのURL",
-        "jobTitle": "職名",
+        "jobTitle": "役職",
```
Comment (Ania-chan) [inline web/src/i18n/locales/ja.json:76]: Is 役職 also appropriate for normal 会社員 jibs that don't have any special title? I changed it to 職名 before to make it more broad. But I'm also not a native speaker so I'd like to know how it sounds in japanese :)

File: web/src/i18n/locales/ja.json [hunk 4]
```diff
@@ -83,10 +83,10 @@
         "submitterEmail": "推薦者のEメール",
         "submitterName": "推薦者",
         "thanks": "ご投稿ありがとうございます！",
-        "title": "スピーカーを指名する",
+        "title": "講演者を指名する",
         "topics": {
             "label": "関心のある分野",
-            "hint": "何かを入力してEnterキーを押し、新しいトピックを追加します"
+            "hint": "分野を入力後、Enterキーを押し、新しい分野を追加します"
```
Comment (Ania-chan) [inline web/src/i18n/locales/ja.json:89]: if we change it to 分野 maybe we can write it like `分野を選択後` ? not to make impression that you need to write the whole sentence- you can just select them from the dropdown

**Review by Ania-chan — COMMENTED** [review]
@shokokb Thank you for your PR and all the translation changes! Overall looks great 🙆‍♀️  but I left you some questions about small details. Let me know if anything is unclear 😸

**Comment by ann-kilzer** [comment]
Hi @shokokb, thanks for your great work on this ticket. I'd love to get some of these changes in so that our users can get the best translation. Would it be okay if one of us edits this branch and brings this over the finish line?

**Comment by ann-kilzer** [comment]
It would be better to continue working on this PR as there is a lot of important discussion here that gets lost when we open new PRs. If you are having trouble with git, I can help you update it with master.

I'd like to re-open this PR, but we need to close #199 and #197 and apply those changes here.

**Pull request closed without merging**